### PR TITLE
Added Readme File to Basic Background Generator

### PR DESCRIPTION
**Pull request by mrkc2303**
I have added the Readme file to Basic Background Generator Project.
This issue fixes : #62 

Kindly review this PR.

### CODE REVIEW

**Review by harshita2216 — CHANGES_REQUESTED** [review]
Looks fine, but it'll be nice if you could just elaborate on the steps a little more. For example, instead of writing go to the directory, you can mention different ways to run it, like directly opening it on Chrome or Live Server using VS Code, etc. Also, mention the use of this project as a separate section.

**Comment by mrkc2303** [comment]
Done!
Please Check

File: Basic Background Generator/Readme.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- Download or clone the repository
+
+```
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+- Go to the directory
+- Run the index.html file in your browser
+- Start creating your own gradient background
+<br>
+## Use of the Project
```
Comment (chicken-biryani) [inline Basic Background Generator/Readme.md:25]: The "Use of the project" isn't getting displayed properly.

File: Basic Background Generator/Readme.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+This project aims to find the best looking background gradient for your project. You just have to select the colors you want from the color picker and then copy the code displayed and paste into your code as background color.
+It's That Simple!!
+<br><br>
+
+![image](https://user-images.githubusercontent.com/67221487/124392353-ab6b0700-dd12-11eb-9093-b6309b08e2d5.png)
+<br>
+![image](https://user-images.githubusercontent.com/67221487/124392372-c2115e00-dd12-11eb-9743-9be785121fb4.png)
```
Comment (chicken-biryani) [inline Basic Background Generator/Readme.md:33]: Use Markdown beautifier for the complete file.
Here use this link :
https://codebeautify.org/markdown-formatter

Comment (mrkc2303) [reply Basic Background Generator/Readme.md]: Changes made, please let me know if any more are required!
@chicken-biryani

**Comment by chicken-biryani** [comment]
Lgtm!

**Pull request merged**